src/repository.ts

```diff
@@ -186,9 +193,13 @@ export class Repository {
       /[\\\/]\.svn[\\\/]/.test(uri.path)
     );
 
+    // TODO on svn switch event fired two times since two files were changed
     onRelevantSvnChange(
-      this._onDidChangeRepository.fire,
-      this._onDidChangeRepository,
+      async (e: Uri) => {
+        await this.repository.updateInfo();
+        this._onDidChangeRepository.fire(e);
+      },
+      this,
```

Comment:
```suggestion
      undefined,
```
this is not mandatory since lambda doesn't have it's own this